### PR TITLE
Matandobr activity launcher

### PR DESCRIPTION
**Pull request by ajinabraham**
### Describe the Pull Request

```
Feature added by @matandobr 
Tested only on windows for now!

Added activity launcher for dynamic execution
Added configurable sleep in activity tester
Disable dynamic report buttons when does not exist
Fixed broken link in setup.bat
Small typos fixes
Added REST API and docs
```

### Checklist for PR

- [x] Run MobSF unit tests and lint `tox -e lint,test`
- [x] Tested Working on Linux, Mac, Windows, and Docker
- [x] Add unit test for any new Web API (Refer: `StaticAnalyzer/tests.py`)
- [x] Make sure tests are passing on your PR [MobSF tests](https://github.com/MobSF/Mobile-Security-Framework-MobSF/actions)

### Additional Comments (if any)

```
DESCRIBE HERE
```

### CODE REVIEW

**Comment by ajinabraham** [comment]
Closing this in favour of https://github.com/MobSF/Mobile-Security-Framework-MobSF/pull/1868

File: mobsf/DynamicAnalyzer/views/android/environment.py [hunk 1]
```diff
@@ -509,7 +509,11 @@ def launch_n_capture(self, package, activity, outfile):
                           'start',
                           '-n',
                           package + '/' + activity], True)
-        self.wait(settings.ACTIVITY_TESTER_SLEEP)
+        if not getattr(settings, 'ACTIVITY_TESTER_SLEEP'):
```
Comment (matandobr) [inline mobsf/DynamicAnalyzer/views/android/environment.py]: @ajinabraham Are you thinking of an option where a user pulled the latest version, but the settings.py wasn't updated? 
I mean, if he updated the the project, both the activity tester + the settings should be updated, or am I missing something

Comment (ajinabraham) [reply mobsf/DynamicAnalyzer/views/android/environment.py]: Not his `settings.py` but`~/.MobSF/config.py` will not be updated which Django uses and will result in an exception.

**Comment by ajinabraham** [comment]
@matandobr I guess this is good to merge.

**Review by matandobr — APPROVED** [review]

**Comment by matandobr** [comment]
@ajinabraham Lets do it :)

**Pull request merged**